ConcatTable.lua

```diff
@@ -85,19 +85,13 @@ end
 function ConcatTable:accGradParameters(input, gradOutput, scale)
    scale = scale or 1
    for i,module in ipairs(self.modules) do
-      module:accGradParameters(input, gradOutput[i], scale)
+      self:callAndRethrow(module, i, 'accGradParameters', input, gradOutput[i], scale)
    end
 end
 
 function ConcatTable:accUpdateGradParameters(input, gradOutput, lr)
    for i,module in ipairs(self.modules) do
-      module:accUpdateGradParameters(input, gradOutput[i], lr)
-   end
-end
-
-function ConcatTable:zeroGradParameters()
```

Comment:
I have included it in the notes below the commit title. I though it's such a small thing that it doesn't need to be mentioned in the title/committed separately.